import-service/src/libs/s3-functions.ts

```diff
@@ -0,0 +1,65 @@
+import { S3Client, DeleteObjectCommand, CopyObjectCommand } from "@aws-sdk/client-s3";
+
+const csv = require('csv-parser');
+
+const parseFile = async (S3, params) => {
```

Comment:
I think this `async` keyword is unused here

Reply:
I removed.